app/controllers/katello/api/v2/host_collections_controller.rb

```diff
@@ -149,6 +155,89 @@ def copy
       respond_for_create :resource => new_host_collection
     end
 
+    api :PUT, "/host_collections/:id/autoattach_subscriptions", N_("List summary of subscriptions on all content hosts in collection")
+    param :id, :identifier, :desc => N_("ID of the host collection"), :required => true
+    def autoattach_subscriptions
+      systems = Katello::System.readable
```

Comment:
If this is making changes to content hosts, would we want to allow users to make that change to content hosts that they can only read?